content/documentation/known-issues/index.adoc

```diff
@@ -28,16 +28,32 @@ No version of Internet Explorer is supported with Kiali. Users may experience so
-
-icon:info[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} JIRA link: https://issues.jboss.org/browse/KIALI-1284[KIALI-1284]
-
 == Graph badly laid out
 
 If you search in the https://groups.google.com/forum/#!msg/kiali-dev/[mailing list], you will find a number of discussions about the graph layouts.
 
 The layout for Kiali Graph may render differently, depending on the data to display (number of graph nodes and their interactions) and it's sometimes difficult, not to say impossible, to have a single layout that renders nicely in every situations. That's why Kiali offers a choice of several layout algorithms. However, we may still find some scenarios where none of the proposed algorithm offer a satisfying display. If Kiali doesn't not render your graph layout in a satisfactory manner please switch to another layout option. This can be done from the _Graph Settings_ dropdown by changing the _Layout Schema_.
 
 icon:info[size=2x] {nbsp}{nbsp}{nbsp}{nbsp} If Kiali doesn't produce a good graph for you, don't hesitate to https://github.com/kiali/kiali/issues/new?template=bug_report.md[open an issue in GitHub] or reach us in the https://groups.google.com/forum/#!forum/kiali-users[mailing lists], and describe your situation.
+
+== Many "unknown" connections in the graph
+
+In some situations you can see a lot of connections from an "Unknown" node to your services in the graph, because some software external to your mesh must be periodically pinging or fetching data. This is typically the case when you setup Kubernetes liveness probes, or have some application metrics pushed or exposed to a monitoring system such as Prometheus. Perhaps you wouldn't like to see these connections because they make the graph harder to read.
+
+There are several ways to prevent Istio from gathering this kind of telemetry.
+
+The first is to have these endpoints (like `/health` or `/metrics`) exposed on a different port and server than your main application, and to _not_ declare this port in your _Pod_'s container definition as _containerPort_. This way, the requests will be completely ignored by the Istio proxy, as mentioned in link:https://istio.io/docs/tasks/traffic-management/app-health-check/#liveness-and-readiness-probes-with-http-request-option[Istio documentation] (at the bottom of that page).
+
+The second way is to modify Istio's Prometheus rule to explicitly exclude some requests based on the User Agent. This is the `Rule` resource named `promhttp` located in `istio-system`. To edit it:
+
+```bash
+kubectl edit rule promhttp -n istio-system
+```
+
+Then locate the `match` field under `spec` section. Change it to filter out, for instance, the Kubernetes probes:
+
+```yaml
+match: (context.protocol == "http" || context.protocol == "grpc") && (match((request.useragent | "-"), "kube-probe*") == false)
+```
+
+Note that, starting from Istio 1.1, Kubernetes probes will be filtered out in that way by default. But you may still want to do something equivalent with other User Agents.
```

Comment:
any docs on Istio website confirming this?

Reply:
No doc afaik, but a PR : https://github.com/istio/istio/pull/10480

Reply:
would be good to mention that PR then :)

Reply:
Link added!